Common/Securities/SecurityManager.cs

```diff
@@ -412,6 +432,46 @@ protected virtual void OnCollectionChanged(NotifyCollectionChangedEventArgs chan
         }
 
         /// <summary>
+        /// Get the data feed type for the given security
+        /// </summary>
+        /// <param name="factoryType"><see cref="BaseData"/> type of the security</param>
+        /// <param name="tickType">The <see cref="TickType"/> of the security</param>
+        /// <returns>Type that should be added as a subscription</returns>
+        private static Type GetDataFeedType(Type factoryType, TickType tickType)
+        {
+            Type dataFeedType;
+
+            // if it is a standard QC Data type
+            if (factoryType == typeof(TradeBar) ||
+                factoryType == typeof(QuoteBar) ||
+                factoryType == typeof(OpenInterest))
+            {
```

Comment:
Good question. @andrewhart098 is this by design? We also have to make sure zip entries are processed correctly.